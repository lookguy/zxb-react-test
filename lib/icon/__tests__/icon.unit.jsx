import Icon from "../icon";
import * as renderer from 'react-test-renderer';
import React from 'react';

describe('icon', () => {
    it('icon xxx', () => {
        const json = renderer.create(<Icon />).toJSON();
        expect(json).toMatchSnapshot();
    });
});