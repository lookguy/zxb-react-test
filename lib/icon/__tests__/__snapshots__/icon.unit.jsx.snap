// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon render icon successful 1`] = `
<svg
  className="zui-icon"
>
  <use
    xlinkHref="#alipay"
  />
</svg>
`;
